docs/getting_started.md

```diff
@@ -24,7 +30,7 @@ the appropriate options and continue.
 [![Assignment](assignment_form.png)](assignment_form.png)
 
 Next you'll be asked to upload your autograder. Upload a zip file
-meeting the required [specifications](specs), select the [base image](base_image)
+meeting the required [specifications](../specs), select the [base image](../base_images)
```

Comment:
We need to fix the relative links. Otherwise they 404 due to the change in https://github.com/mkdocs/mkdocs/issues/292